guides/doc-Configuring_virt-who_VM_Subscriptions/master.adoc

```diff
@@ -0,0 +1,95 @@
+:context: virt-who-vm-subs
+:virt-who-vm-subs:
+:ProductName: {ProjectServer}
+
+= {ConfiguringVMSubscriptionsDocTitle}
+
+ifdef::foreman-deb[]
+You cannot use the virt-who plugin because it requires the Katello plugin, which is not supported on Debian or Ubuntu.
+endif::[]
+ifdef::foreman-el[]
+You cannot use the virt-who plugin without the Katello plugin.
+endif::[]
+
+ifdef::satellite[]
+include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
+endif::[]
+
+ifdef::katello,satellite,orcharhino[]
+include::common/assembly_overview-of-vm-subscriptions.adoc[leveloffset=+1]
+
+// To do
+// include::common/modules/ref_virt-who-configuration-examples.adoc[leveloffset=+2]
+
+:!virt-who-vm-subs:
+:context: kvm
+:kvm:
+include::common/modules/con_configuring-virt-who-for-provider.adoc[leveloffset=+1]
+
+include::common/modules/proc_creating-a-virt-who-configuration.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-hypervisor-smartproxy.adoc[leveloffset=+2]
+:!kvm:
+:context: vmware
+:vmware:
+include::common/modules/con_configuring-virt-who-for-provider.adoc[leveloffset=+1]
+
+include::common/modules/proc_creating-a-virt-who-configuration.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-projectserver.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-hypervisor-smartproxy.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-el.adoc[leveloffset=+2]
+:!vmware:
+:context: nutanix
+:nutanix:
+include::common/modules/con_configuring-virt-who-for-provider.adoc[leveloffset=+1]
+
+include::common/modules/proc_creating-a-virt-who-configuration.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-projectserver.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-hypervisor-smartproxy.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-el.adoc[leveloffset=+2]
+:!nutanix:
+:context: hyperv
+:hyperv:
+include::common/modules/con_configuring-virt-who-for-provider.adoc[leveloffset=+1]
+
+include::common/modules/proc_creating-a-virt-who-configuration.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-projectserver.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-hypervisor-smartproxy.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-el.adoc[leveloffset=+2]
+:!hyperv:
+:context: openstack
+:openstack:
+include::common/modules/con_configuring-virt-who-for-provider.adoc[leveloffset=+1]
+
+include::common/modules/proc_creating-a-virt-who-configuration.adoc[leveloffset=+2]
+
+include::common/modules/proc_deploying-a-virt-who-configuration-on-hypervisor-smartproxy.adoc[leveloffset=+2]
+:!openstack:
+:context: kubevirt
+:kubevirt:
```

Comment:
Here is where you are erasing the KubeVirt's attribute value. Before line 80, the value is taken from attributes-*.adoc. After line 80, the value is blank.

Reply:
Right, adoc is case insensitive. That bastard.

Reply:
Looks like the solution is to find a context that is not the same as the attribute! Thanks for testing this!!!